kernel/panic.c

```diff
@@ -373,7 +373,23 @@ void panic(const char *fmt, ...)
 	if (len && buf[len - 1] == '\n')
 		buf[len - 1] = '\0';
 
-	pr_emerg("Kernel panic - not syncing: %s\n", buf);
+	pr_emerg(
+		"⡧⣾⣿⡿⣫⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣟⣿⣿⣿⣿⣷⣿⣻⣿⣮⡻⣿⣿⣿⣿\n"
```

Comment:
How exactly is this related to panic? Needs a comment with an explanation.
Need to provide information about the origin of this image and a copyright notice.
Braille might not display well in the serial console.

Reply:
The "not syncing" message (whatever it means) was lost in this commit.

Reply:
this pr could be so clear without deletions...

Reply:
With DRM kernel panic you could display anime series about different error causes